developers_chamber/scripts/project.py

```diff
@@ -184,6 +188,19 @@ def install(project_name, compose_file, var_dir, container_dir_to_copy, install_
     compose_install(project_name, compose_file, var_dir, container_dir_to_copy, install_container_command)
 
 
+
+@project.command()
+@click.option('--library-source-dir', '-s', help='Library source directory', type=str, required=True)
+@click.option('--library-destination-dir', '-d', help='Library destination directory', type=str, required=True,
+              default=default_library_dir)
+def bind_library(library_source_dir, library_destination_dir):
```

Comment:
celkem prosté 

bind-library ../libs/django-is-core/core 

a je to. Nic se nemusí přeinstalovávat

Reply:
The command accepts two args...

Reply:
sorry 

pydev project bind-library -s ../libs/django-is-core/core

the second one is defined in project settings but you can use

pydev project bind-library -s ../libs/django-is-core/core -d var/site-packages